.github/workflows/benchmarks.yml

```
@@ -29,7 +29,7 @@ jobs:
         id: go
 
       - name: Run benchmark
-        run: go test  -benchmem -run="^$" -bench "^Benchmark" . github.com/timbray/quamina | tee output.txt
+        run: go test  -benchmem -run="^$" -bench "^Benchmark" . quamina.net/go/quamina | tee output.txt
```

Question: do we need the go mod path here at all? I think this is enough:

```suggestion
        run: go test  -benchmem -run="^$" -bench "^Benchmark" . | tee output.txt
```

Or do you remember why the module path was added there?

---

No idea, sorry. This was done by @yosi I think?

---

I want to get this merged so I think I'll leave this in for now post a question to @yosiat and see if we can take this out at some future point

---

(I mean, the benchmarks tests are passing and previously they weren't so I want to be careful)